apps/epic-web/src/inngest/functions/stripe/slack-daily-reporter.ts

```diff
@@ -255,53 +327,105 @@ export const slackDailyReporter = inngest.createFunction(
                 '#4BC0C0',
                 '#9966FF',
                 '#FF9F40',
+                '#FF0000',
+                '#008B8B',
+                '#8B4513',
+                '#FFFF00',
+                '#008080',
+                '#FF00FF',
+                '#800000',
+                '#00FFFF',
+                '#006400',
+                '#FFC0CB',
+                '#808000',
+                '#4B0082',
+                '#A52A2A',
+                '#40E0D0',
               ],
+              maxBarThickness: 50,
+              minBarLength: 2,
             },
           ],
         },
         options: {
-          title: {
-            display: true,
-            text: 'Revenue Distribution by Product',
-            fontColor: 'black',
-          },
-          legend: {
-            position: 'bottom',
-            labels: {
-              boxWidth: 15,
-              padding: 15,
-              fontColor: 'black',
-            },
-          },
+          indexAxis: 'y',
           plugins: {
+            legend: {
+              display: false,
+            },
+            title: {
+              display: true,
+              text: `Revenue Breakdown by Product for ${currentMonth} to Date (Excluding Expenses)`,
+              color: 'black',
+              font: {
+                size: 10,
+              },
+            },
+            subtitle: {
+              display: true,
+              text: 'Estimated Royalty: ' + formattedTotal,
+              color: 'black',
+              font: {
+                size: 10,
+                weight: 'bold',
+              },
+              padding: {
+                bottom: 10,
+              },
+            },
             datalabels: {
+              formatter: function (value: any) {
```

Comment:
I'm struggling to make this loads! It seems to be a bug in quickchart, for now it displays as a number format and no the currency 

https://chartjs-plugin-datalabels.netlify.app/guide/formatting.html#data-transformation